Reqnroll/TestRunnerManager.cs

```diff
@@ -256,20 +302,19 @@ public static async Task OnTestRunEndAsync(Assembly testAssembly = null, IContai
         }
     }
 
-    public static async Task OnTestRunStartAsync(Assembly testAssembly = null, string testWorkerId = null, IContainerBuilder containerBuilder = null)
+    public static async Task OnTestRunStartAsync(Assembly testAssembly = null, IContainerBuilder containerBuilder = null)
     {
         testAssembly ??= GetCallingAssembly();
         var testRunnerManager = GetTestRunnerManager(testAssembly, createIfMissing: true, containerBuilder: containerBuilder);
-        testRunnerManager.GetTestRunner(testWorkerId ?? TestRunStartWorkerId);
```

Comment:
Can we add a test for it that is currently failing?
Alternative we could create the Testrunner (as before) and instantly release it (new).

Reply:
It would be great to have such a test. I can try next week.
(This is a tricky problem, because if the binding registry is not initialized at the right time, it will cause problems only with parallel runs.)

Reply:
If this is the last open issue with the pr, we can still get the TestRunner and release it immediately. The topic (why is this being done) was there before.

Reply:
@obligaron There is no need for that. I have checked it and found:
* there are already such tests in `TestRunnerManagerRunnerCreationTests`
* they worked before and still work
* I merged them to `TestRunnerManagerTests` for simplicity (see last commit of the PR)